### PR TITLE
Add ability to highlight a specific choice from the list of choices

### PR DESCRIPTION
**Pull request by josephschmitt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngc-omnibox",
-  "version": "0.2.2",
+  "version": "0.2.3",
```
Comment (onlywei) [inline package.json]: This is a new "feature", so technically you should be incrementing the middle number.

File: spec/tests/angularComponent/ngcOmniboxControllerSpec.js [hunk 2]
```diff
@@ -446,6 +446,24 @@ describe('ngcOmnibox.angularComponent.ngcOmniboxController', () => {
...
+
+      omniboxController.highlightChoice('two');
+      expect(omniboxController.highlightedChoice).toBe('two');
+    });
+
+    it('should only highlight a choice that exists in the list of choices', () => {
+      omniboxController.highlightChoice('two');
+      expect(omniboxController.highlightedChoice).toBe('two');
+
+      omniboxController.highlightChoice('four');
```
Comment (onlywei) [inline spec/tests/angularComponent/ngcOmniboxControllerSpec.js]: Maybe change 'four' to 'bogus'? It's not clear to me reading this diff that "four" is not in the list of choices. I have to do a lot of code expanding.

File: package.json (hunk 1 above)
Comment (josephschmitt) [inline package.json]: Done

File: spec/tests/angularComponent/ngcOmniboxControllerSpec.js (hunk 2 above)
Comment (josephschmitt) [inline spec/tests/angularComponent/ngcOmniboxControllerSpec.js]: Done

**Pull request merged**